packages/connect-disputable-voting/src/models/ERC20.ts

```diff
@@ -1,15 +1,34 @@
+import { miniMeAbi } from '@aragon/connect-core'
+import { BigNumber, Contract, providers as ethersProviders } from 'ethers'
+
 import { ERC20Data } from '../types'
 
 export default class ERC20 {
+  #provider: ethersProviders.Provider
+
   readonly id: string
   readonly name: string
   readonly symbol: string
   readonly decimals: string
 
-  constructor(data: ERC20Data) {
+  constructor(data: ERC20Data, provider: ethersProviders.Provider) {
+    this.#provider = provider
+
     this.id = data.id
     this.name = data.name
     this.symbol = data.symbol
     this.decimals = data.decimals
   }
+
+  async getBalance(address: string): Promise<BigNumber> {
+    return this._contract().balanceOf(address)
+  }
```

Comment:
Should we name this method `balance()`? Not for this PR, but I am also thinking that it could be nice to have a `onBalance()` here (by querying it on the block event).